www/include/views/graphs/getGraphAjax.php

```diff
@@ -34,7 +34,7 @@
 
 require_once realpath(dirname(__FILE__) . '/../../../../config/centreon.config.php');
 
-require_once _CENTREON_PATH_ . '/www/class/centreonDB.class.php';
+require_once _CENTREON_PATH_ . "/bootstrap.php";
```

Comment:
It's same

This code is functional, but to improve the code, it's better to remove the line 
`require_once realpath(dirname(__FILE__) . "/../../../../../config/centreon.config.php");`
And change the path of require "bootstrap.php" whith the relative path.
Because bootstrap.php allready load the configurations files.